app/routes/account/billing/invoices/list.js

```diff
@@ -59,7 +59,7 @@ export default class extends Route.extend(EmberTableRouteMixin) {
 
     queryString = this.applySortFilters(queryString, params);
     return {
-      eventInvoices: (await this.store.query('event-invoice', queryString)).toArray(),
+      eventInvoices: await this.asArray(this.store.query('event-invoice', queryString)),
```

Comment:
Are the event invoices being returned from the model properly here?

Reply:
Yes